src/model/Book.js

```diff
@@ -0,0 +1,18 @@
+import mongoose from 'mongoose';
+
+const Schema = new mongoose.Schema({
```

Comment:
can we improve our book schema?
add ISBN \o/

Reply:
done, ISBN added.